download/download.adoc

```diff
@@ -66,7 +66,7 @@ A CR should be almost identical to the Final release.
 * image:download.png[] Download jBPM *#{site.pom.latest.version}* binaries
 ** #{site.pom.latest.jbpmBinZip}[Download jBPM #{site.pom.latest.version}-bin.zip]
 ** #{site.pom.latest.jbpmExamplesZip}[Download jBPM #{site.pom.latest.version}-examples.zip]
-** #{site.pom.latest.jbpmInstallerZip}[Download jBPM #{site.pom.latest.version}-installer.zip]
+** #{site.pom.latest.jbpmInstallerZip}[Download jBPM installer-#{site.pom.latest.version}.zip]
```

Comment:
@psiroky I saw when uploading the binaries to filemgmt.jboss.org that the jbpm-installer changes its name: from jbpm-<version>-installer.zip to jbpm-installer-<version>.zip - so I thought maybe they want to have this file another name